src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -139,14 +139,16 @@ bool emberAfGeneralCommissioningClusterArmFailSafeCallback(app::CommandHandler *
     Commands::ArmFailSafeResponse::Type response;
 
     /*
+     * If the fail-safe timer is in processing, then the fail-safe timer SHALL not be armed.
```

Comment:
This looks like a normative claim from the spec... but I see nothing like this in the spec.  Where is it coming from?

How about:
```suggestion
     * If the fail-safe timer is not fully disarmed, don't allow arming a new fail-safe.
```